modules/guides/pages/kv-operations.adoc

```diff
@@ -0,0 +1,350 @@
+= Key Value Operations
+:description: How to perform CRUD key value operations in Couchbase.
+:page-pagination: next
+:page-topic-type: guide
+:page-toclevels: 2
+:tabs:
+:github: Click the GitHub button icon:github[] to view this code in context.
+
+include::partial$example-attributes.adoc[]
+
+[abstract]
+{description}
+
+== Introduction
+
+Every item in a database goes through the basic _CRUD_ cycle, which is typical of an application’s use of data.
+CRUD stands for create, read, update, and delete:
+
+* **C**reate: when data is first inserted into the cluster
+* **R**ead: when an application retrieves the data
+* **U**pdate: when data is modified to reflect a change in the state represented by the data
+* **D**elete: when the data is no longer needed
+
+The xref:learn:services-and-indexes/services/data-service.adoc[Key Value (KV) or Data Service] offers Couchbase clients the fastest and simplest way to create, retrieve or mutate data where the key is known.
+
+include::partial$before-you-begin.adoc[]
+
+include::partial$clients.adoc[]
+
+== Creating Documents
+
+Creating a document means inserting new data into the database.
+To create a single document in Couchbase, perform an `insert` operation.
+
+[{tabs}]
+====
+cbc::
++ 
+-- 
+To perform an insert operation, use the `cbc create` command with `--mode insert`.
+
+{kv-simple-insert-example}
+
+.airport-123.json
+[source,json]
+----
+include::example$kv/airport-123.json[]
+----
+
+[source,shell]
+----
+include::example$kv/kv-cbc.sh[tag=cbc-simple-insert]
+----
+--
+
+.NET::
++ 
+--
+{kv-simple-insert-example}
+
+[source,csharp]
+----
+include::dotnet-sdk:hello-world:example$KvHelloWorldDefault.csx[tag=kv-simple-insert,indent=0]
+----
+
+{github}
+--
+
+Java::
++ 
+--
+{kv-simple-insert-example}
+
+[source,java]
+----
+include::java-sdk:hello-world:example$KvHelloWorldDefault.java[tag=kv-simple-insert,indent=0]
+----
+
+{github}
+--
+
+Node.js::
++ 
+--
+{kv-simple-insert-example}
+
+[source,nodejs]
+----
+include::nodejs-sdk:hello-world:example$kv-hello-world-default.js[tag=kv-simple-insert,indent=0]
+----
+
+{github}
+--
+
+Python::
++ 
+--
+{kv-simple-insert-example}
+
+[source,python]
+----
+include::python-sdk:hello-world:example$kv_hello_world_default.py[tag=kv-simple-insert,indent=0]
+----
+
+{github}
+--
+====
```

Comment:
It's a shame because I quite like the idea of progressive disclosure, i.e. "here's a snippet of what you'll see on the following pages". But on reflection I think you should remove this example from this page. The presence of examples on this page will make users think that they've found the actual page with the procedures for KV operations, and not realise (or understand why) there are other pages which give the same procedures with more detail. Sorry about that.